client/lxd.go

```diff
@@ -269,9 +269,6 @@ func (r *ProtocolLXD) rawQuery(method string, url string, data any, ETag string)
 
 			// Set the encoding accordingly
 			req.Header.Set("Content-Type", "application/json")
-
-			// Log the data
-			logger.Debugf(logger.Pretty(data))
```

Comment:
This will make `lxc --debug` much harder to use for API debugging.

Can we do something else here?

Reply:
Mh we could try converting the `any` type back to any of the structs that contain passwords and obfuscate the field with `***`?

Reply:
Lets leave it for now.

Reply:
or perhaps rather than using logger.Debugf we could print to stderr (which is what the logger.Debugf does when using `--debug` anyway right)?